.travis.yml

```diff
@@ -0,0 +1,33 @@
+sudo: required
+dist: trusty
+
+language: python
+
+python:
+    - "2.7"
+
+virtualenv:
+    system_site_packages: true
+
+services:
+    - cassandra
```

Comment:
Good point.  I'll give it a try without this line to see if it can still attach to a by-hand allocated cassandra service.  I can't see why it wouldn't.